backend/databases/qdrant.py

```
@@ -0,0 +1,105 @@
+        # Delete existing collection if it exists
+        if collection_exists(QDRANT_COLLECTION_NAME):
+            qdrant_client.delete_collection(collection_name=QDRANT_COLLECTION_NAME)
+            print(f"Deleted existing collection: {QDRANT_COLLECTION_NAME}")
+        
+        # Create new collection with updated vector size
+        qdrant_client.create_collection(
+            collection_name=QDRANT_COLLECTION_NAME,
+            vectors_config=VectorParams(size=VECTOR_SIZE, distance=Distance.COSINE),
+        )
+        print(f"Created new collection: {QDRANT_COLLECTION_NAME} with vector size {VECTOR_SIZE}")
+        return True
```

Ruff (0.16.1)

[warning] Consider moving this statement to an `else` block

(TRY300)